v3/src/components/map/components/inspector-panel/map-inspector.scss

```diff
@@ -1,34 +1,84 @@
 .inspector-panel.map.data-display {
   .codap-inspector-palette {
-    .map-layers-checkbox {
-      font-weight: bold;
+    font-size: 12px;
+    min-width: 205px;
+    min-height: 100px;
+    font-weight: bold;
+
+    .palette-form {
+      height: fit-content;
+
+      .palette-row {
+        height: 30px;
+
+        .form-label.color {
+          margin-right: 4px;
+        }
+      }
+
+      .palette-checkbox {
+        height: 20px;
+        width: 180px;
+        margin: 2px 0;
+      }
+
+      .map-values-controls {
+        display: flex;
+        flex-direction: column;
+        margin: 5px;
+
+        .map-values-layer-label{
+          display: flex;
+          align-items: center;
+        }
+      }
     }
 
     .map-base-control {
       margin: 1px;
 
+      .palette-checkbox{
+        width: 80px;
+      }
+
       .map-base-segmented {
-        border: 1px solid #cccccc;
+        height: 18px;
+        width: 170px;
         margin: 5px;
-        border-radius: 10px;
-        padding-left: 5px;
-        padding-right: 5px;
-        font-size: 12px;
+        border-radius: 4px;
+        justify-content: center;
         cursor: pointer;
 
         .map-base-button {
-          border-radius: 5px;
-          padding: 5px;
-          background-image: linear-gradient(to bottom, #f7f7f7, #e7e7e7);
-        }
-
-        .map-base-button-selected {
-          border-radius: 5px;
-          padding: 5px;
-          background-image: linear-gradient(to bottom, #5DA3EE, #437BD4);
-          color: white;
-          font-weight: bold;
-          text-shadow: black 1px 1px;
+          border-radius: 4px;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          overflow: hidden;
+          &.oceans {
+            border: 1px solid #cccccc;
+            border-radius: 4px 0 0 4px;
+            height: 18px;
```

Comment:
Seems like the height is always `18px`. Can it move up to the `.map-base-button`?